src/styles/03-organisms/list/_list.scss

```diff
@@ -0,0 +1,10 @@
+.unordered-list {
+  @include list-reset;
+
+  &__list-container {
```

Comment:
Looking at the React file this list-container is outside of `.unordered-list`, so these styles aren't ever being applied.

Reply:
changed up the list tsx so I think it's applied now.